packages/dom/src/tabbable.js

```diff
@@ -174,14 +169,17 @@ export function findPrevious( element ) {
 	// Remove all focusables after and including `element`.
 	focusables.length = index;
 
-	return last( filterTabbable( focusables ) );
+	const tabbable = filterTabbable( focusables );
+	return tabbable[ tabbable.length - 1 ];
```

Comment:
Nit: could also use `tabbable.at(-1)` ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at))

Reply:
We could, but TypeScript will complain:

```
 error TS2339: Property 'at' does not exist on type 'Element[]'
```

This is fixable, but definitely not something I want to make part of this PR, which is why I preferred the old-school way.